schema_utils.py

```diff
@@ -179,6 +213,26 @@ def normalize_against_schema(
             assert isinstance(obj, python_utils.UNICODE), (
                 'Expected unicode, received %s' % obj)
         normalized_obj = obj
+    elif schema[SCHEMA_KEY_TYPE] == SCHEMA_TYPE_OBJECT_DICT:
+        # The schema type 'object_dict' accepts either of the keys
+        # 'object_class' or 'validation_method'.
+        # 'object_class' key is the most commonly used case, when the object is
+        # initialized from_dict() method and the validation is done from
+        # validate() method.
+        # 'validation_method' key is used for some rare cases like if they have
+        # validate_dict method instead of validate method, or if they need some
+        # extra flags like for strict validation. The methods are written in the
+        # domain_objects_validator file.
+
+        if SCHEMA_KEY_OBJECT_CLASS in schema:
```

Comment:
Make sure to update the wiki page just after this PR is merged! Have you got reviewers on the draft wikipage doc?

Reply:
Yes, the draft doc of the wiki page is reviewed by Sean and Rohit.